packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -121,12 +110,78 @@ MeetingUtil.getSdps = (meeting) => {
   if (meeting.mediaProperties.mediaPeerConnection) {
     sdps.push(meeting.mediaProperties.mediaPeerConnection.sdp);
   }
-
   if (meeting.mediaProperties.sharePeerConnection) {
     sdps.push(meeting.mediaProperties.sharePeerConnection.sdp);
   }
 
   return sdps;
 };
 
+MeetingUtil.leaveMeeting = (meeting) => meeting.meetingRequest
+  .leaveMeeting({
+    locusUrl: meeting.locusUrl,
+    selfId: meeting.selfId,
+    correlationId: meeting.correlationId,
+    resourceId: null,
+    deviceUrl: meeting.deviceUrl
+  })
+  .then((response) => meeting.locusInfo.onFullLocus(response.body.locus))
+  .then(() => meeting.closeLocalStream())
+  .then(() => meeting.closeLocalShare())
+  .then(() => meeting.closePeerConnections())
+  .then(() => {
+    meeting.unsetLocalStream();
+    meeting.unsetLocalShare();
+    meeting.unsetRemoteStream();
+    meeting.unsetPeerConnections();
+  })
+  .then(() => meeting.roap.stop(meeting.correlationId, meeting.roapSeq))
```

Comment:
Shouldn't we stop roap sooner? Wondering if we will get ROAP events after we've already torn down the media and left